docs/developer-docs/latest/getting-started/quick-start.md

```diff
@@ -1,150 +1,488 @@
+4. Click **Save**.
+5. Go back to _Collection types > Categories_, then click again on **+ Add New Categories**.  
+6. Type `Brunch` in the **Name** field, then click **Save**.
 
-## 6. Add categories to the "Category" Content Type
+![GIF: Add Categories](../assets/quick-start-guide/qsg-handson-categories.gif)
 
-Navigate to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category).
+The `French Food` and `Brunch` categories are now listed in the [_Collection types > Categories_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category) view.
 
-- Click on **+ Add New Categories** button. Type `French Food` in the **Name** field. Select `Biscotte Restaurant`, on the right in the **Restaurant (0)** dropdown.
-- Click **Save**.
+#### Add a Category to a Restaurant
 
-Navigate back to [**COLLECTION TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::category.category). You will see the **French Food** category listed in the entries.
+Go to [_Collection types > Restaurants_](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant) in the left-hand menu, and click on `Biscotte Restaurant`.
 
-- Click on **+ Add New Categories** button. Type `Brunch` in the **Category** field. **DO NOT ADD** `Biscotte Restaurant` to the `Restaurants` dropdown on the right.
-- Click **Save**.
+In the right sidebar, under **Categories (1)**, click on **Add an item…** and select `Brunch`.  Click **Save**.
 
-You will see the **Brunch** category listed in the entries.
+### Step 3: Set Roles & Permissions
 
-Navigate back to [**COLLECTION TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/collectionType/application::restaurant.restaurant).
+1. Click on _General > Settings_ at the bottom of the left-hand menu.
+2. Under _Users & Permissions Plugin_, choose [**Roles**](http://localhost:1337/admin/settings/users-permissions/roles).
+3. Click the **Public** role.
+4. Scroll down under **Permissions**.
+5. In the **Application** tab, find **Restaurant**. 
+6. Click the checkboxes next to **find** and **findone**.
+7. Repeat with **Category**: click the checkboxes next to **find** and **findone**.
+8. Finally, click **Save**.
 
-- Click on `Biscotte Restaurant`.
-- On the right, under **Categories(1)**, `select` the `Add an item...`, and add **Brunch** as a category for this restaurant, and click the **Save** button.
+![Screenshot: Public Role in Users & Permissions plugin](../assets/quick-start-guide/qsg-handson-part2-04-roles.png)
 
-You have now seen **two different ways** to use the **relation** field type to add and connect relations between Content Types.
+### Step 4: Publish the content
 
-## 7. Set Roles and Permissions
+By default, any content you create is saved as a draft. Let's publish our entries in the collection types.
```

Comment:
This `Let's publish our entries in the collection types.` sounds strange for me.

```diff
- Let's publish our entries in the collection types.
+ Let's publish our entries from the Restaurant collection types.
```

Reply:
This step is about publishing *all* entries, restaurants *and* categories. And we actually start with categories ;-)